components/yaml.rst

```diff
@@ -250,6 +250,65 @@ representation of the object.
     parsers will likely not recognize the ``php/object`` tag and non-PHP
     implementations certainly won't - use with discretion!
 
+Syntax Validation
+~~~~~~~~~~~~~~~~~
+
+The syntax of YAML contents can be validated through the CLI using the
+:class:`Symfony\\Component\\Yaml\\Command\\LintCommand` command.
+
+First, install the Console component:
+
+.. code-block:: bash
+
+    $ composer require symfony/console
+
+Create a script containing a console application:
+
+.. code-block:: php
+
+    // lint.php
+
+    use Symfony\Component\Console\Application;
+    use Symfony\Component\Yaml\Command\LintCommand;
+
+    (new Application('yaml/lint'))
+        ->add(new LintCommand())
+        ->getApplication()
+        ->setDefaultCommand('lint:yaml', true)
+        ->run();
+
+Then, execute the script for validating contents:
+
+.. code-block:: bash
+
+    # validates a single file
+    $ php lint.php path/to/file.yml
+
+    # or all the files in a directory
+    $ php lint.php path/to/directory
+
+    # or contents passed to STDIN
+    $ cat path/to/file.yml | php lint.php
+
+The result will be written on STDOUT, default in TXT format:
+
+.. code-block:: bash
+
+    [OK] All 1 YAML files contain valid syntax.
+
+For a JSON output, use the ``format`` option:
+
+.. code-block:: bash
+
+    $ php lint.php path/to/file.yml --format json
+
+    [
+        {
+            "file": "path/to/file.yml",
+            "valid": true
+        }
+    ]
```

Comment:
Do I need something specific for better simulating a console output? 
Maybe just showing the command and removing the previous block showing only the default output could be enough.